test/jdk/java/awt/Focus/UnaccessibleChoice/AccessibleChoiceTest.java

```diff
@@ -142,7 +154,7 @@ public void start () throws IOException {
         robot.delay(1000);
 
         // On success second item should be selected
-        if (choice.getSelectedItem() != choice.getItem(1)) {
+        if (!choice.getSelectedItem().equals(choice.getItem(1))) {
             // Print out os name to check if mac conditional is relevant
             System.err.println("Failed on os: " + osName);
```

Comment:
Not strictly required to print osName in case of failure (since the details are available), okay to have it though.